neuralmonkey/decoders/monster_decoder.py

```diff
@@ -23,7 +23,7 @@
 class Decoder(object):
```

Comment:
To je divný, já ho tam měl pořád. :-) Nechal bych ho tam dokud nereplikujem výsledky, kterých se s nim dosáhlo.. :-) Ale je fakt, že ta nejdůležitější část je ta decoding_function, kterou už mam vyrefaktorovanou v jiný branchi...